pandas/core/groupby/groupby.py

```diff
@@ -403,7 +403,10 @@ class providing the base-class of operations.
 f : function, str
     Function to apply to each group. See the Notes section below for requirements.
 
-    Can also accept a Numba JIT function with
+    Accepted inputs are:
+
+    - A String
+    - Numba JIT function with
     ``engine='numba'`` specified.
```

Comment:
I think you have to add a blank line here

Reply:
I think that this line needed to be a part of the line above and not on its own separate line, so I moved it up. If that is incorrect, then I apologize and I will make that change. This has been changed here: 3b84281aea5673b2e3ba4624e2b7532895c2860a